app/assets/javascripts/listings/components/before-applying-list.html.slim

```diff
@@ -0,0 +1,6 @@
+ol.numbered-list
```

Comment:
Since there isn't a component associated with this, this file is a partial and should go under 

app/javascripts/listings/templates/_before-applying-list.html.slim